desktop/package.json

```diff
@@ -11,11 +11,10 @@
 	},
 	"license": "Apache-2.0",
 	"scripts": {
-		"start": "pnpm run build-ts && cross-env NODE_ENV=production electron .",
-		"start:dev": "pnpm run build-ts && cross-env NODE_ENV=development electron .",
-		"build-ts": "tsc -p tsconfig.json",
-		"build-electron": "pnpm run build-ts && electron-builder --publish never",
```

Comment:
Is there a specific reason why this script was deleted?

Reply:
I think my question was unclear.

Is there a reason why the `build-electron` script was removed? There doesn't seem to be anything in the changes that can replace the command for building an electron app without publishing.